colossalai/booster/plugin/hybrid_parallel_plugin.py

```diff
@@ -376,22 +380,39 @@ def __init__(
         self.stage_manager = None
         self.schedule = None
         self.custom_policy = custom_policy
+        self.num_microbatches = num_microbatches
```

Comment:
It seems that this attribute is not used.

Reply:
I followed 1f1b here https://github.com/hpcaitech/ColossalAI/blob/8aed02b957082e676cf424068583c78240b27f9d/colossalai/pipeline/schedule/one_f_one_b.py#L42 user can pass in either microbatch_size  or num_microbatches.

Reply:
This is stored in schedule. Why store here again?

Reply:
I see. Removed from plugin attr